src/components/HistoryTables/MessageHistory/History/MessageConfirmedRow.tsx

```diff
@@ -33,17 +25,17 @@ export default function MessageHistoryRow(props: MessageHistoryRowProps) {
     () => isAddrEqual(message.to, inspectingAddress),
     [message.to, inspectingAddress]
   )
-  const { methodName } = useMethodName(message)
+  const methodName = useMethodName(message.to, message.method)
   const { age } = useAge(message?.height)
 
   return (
-    <MessageHistoryRowWrapper>
+    <tr>
       <td>
         <SmartLink href={cidHref(message.cid)}>
           {truncateAddress(message.cid)}
         </SmartLink>
       </td>
-      <td className='capitalize'>{methodName}</td>
+      <td>{methodName}</td>
```

Comment:
methodName is now returned as e.g. `AddSigner`. We can add a regex to split these words so it says `Add Signer`, but I'd propose to do that in `filecoin-actor-utils`

Reply:
Low priority